tests/typescript/conformance/types/any/anyAsConstructor.ts

```diff
@@ -0,0 +1,10 @@
+// any is considered an untyped function call
+// can be called except with type arguments which is an error
+
+var x: any;
+var a = new x();
+var b = new x('hello');
+var c = new x(x);
+
+// grammar allows this for constructors
+var d = new x<any>(x); // no error
```

Comment:
you want to add an empty line at the end of the files so that the little red icon doesn't appear :)